config/tips/marinetips.txt

```diff
@@ -0,0 +1,54 @@
+As a doctor, surgery time from fastest to slowest is: by hand, Autodoc manual, Autodoc automatic.
+As a doctor, the white vest fits on your scrubs and can hold surgical tools.
+As a doctor, wear the surgical apron to help prevent infection while on the ground.
+As a doctor, you've got access to the chemistry machine, experiment to find the best medications.
+As a doctor, hygiene is important! keep yourself clean to prevent infections.
+As an engineer, you can take sandbag barricades apart with your entrenching tool.
+As an engineer, you can repair barricades with a welder.
+As an engineer, C4 is cheaper but det packs are more versatile.
+As an engineer, det packs can be set on demolition mode for an explosive trap.
+As an engineer, you build barricades the fastest.
+As a medic, the medevac stretcher needs to be linked to an active, powered beacon for it to teleport the patient.
+As a medic, the marines are counting on you to fix them up.
+As a medic, scan before you treat and you can't be beat.
+As a medic or doctor, remember that your HUD will tell you how much time is left to defibrillate a patient.
+As a medic, remember to bring extra kelotane or dermaline to heal burn damage. Xenos love acid and burning marines.
+As a B18 specialist, your armor has a built in medical system.
+As a scout specialist, you can take your battle rifle or your custom shotgun.
+As a sniper specialist, you've got a tarp that can make you stealth so long as you don't move.
+As a pyro specialist, your flamethrower has an extinguisher built into it.
+As a human, you can do almost anything if you put your mind to it, it just takes time.
+As a PFC, you can take two guns, one in the suit slot and one on your back.
+As a PFC, you can help out your squad by bringing extra ammunition and food to share.
+As a PFC, you can usually get two attachments of your choice from Requisitions.
+Hypervene is useful. Use it to purge deadly neurotoxins and larva boosting growth hormones.
+As the RO, things get more expensive the more often you buy it.
+While medics and doctors are best, any human can use a Defibrillator.
+Even if a patient dies a second after being successfully revived by a defibrillator, this resets the time it takes for them to become permanently brain dead.
+Any human can make razor wire with four metal rods and barbed wire from metal sheets.
+Using a multitool on a lasgun batteries will turn it into an improvised explosive with a random timer.
+In a pinch, you can use lasgun batteries to power APCs and sentry guns.
+Lasgun batteries can be recharged in APCs and power cell rechargers if the generators are on.
+As a marine, if you are not a PFC, then you have a special room in prep with special gear vendors.
+AP bullets are not always better, they do less damage than normal but penetrate more armor.
+Did you know, that pistols are not complete garbage?
+There is no �best� loadout. Experiment and find what works for you.
+Dying is part of the game.
```

Comment:
This tip is meta. Please curate this section for things that would make sense IC.
Most of these tips are straight out player-to-player ones. Try to conceal them with IC parlance.